src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
+.form {
+  margin-bottom: 20px;
+}
+
+.form > .form-field:not(:last-child) {
```

Comment:
```suggestion
.form-field:not(:last-child) {
```
do not use combined selectors, just use classes